### DISCUSSION TITLE
WixStandardBootstrapperApplication overridable variables not working?

### DISCUSSION THREAD

**Discussion by martinh2011** [wixtoolset/issues#8764] [Questions]
I have a bundle using WixStandardBootstrapperApplication (wix toolset v5) which defines a variable like this:

  <Variable Name="Foo"
            Type="string"
            Value="bar"
            Persisted="yes"
            bal:Overridable="yes" />
            
 When I run the bundle with 
 
 bundlesetup.exe Foo=test 
 
 I get 
 
 Ignoring attempt to set non-overridable variable: 'Foo'.
 
 in the log.
 
 What am I doing wrong?

**Comment by wmanning** [comment]
The variable has to be in burn and marked as Overridable.

**Reply by wmanning** [reply]
https://stackoverflow.com/questions/14762617/wix-bootstrapper-how-do-i-set-burn-variables-from-the-command-line
